tests/manual-test-cases/Group20-Security/20-1-Appliance-Audit.robot

```diff
@@ -0,0 +1,56 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 20-01 - Appliance-Audit
+Suite Setup  Setup Test Environment
+Suite Teardown  Clean up VIC Appliance And Local Binary
+Resource  ../../resources/Util.robot
+
+*** Keywords ***
+Setup Test Environment
+    [Arguments]  ${version}=
```

Comment:
Can you make the default at least ${EMPTY} to be explicit, I am not really sure what the behavior of leaving it like this.